runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java

```diff
@@ -657,6 +698,13 @@ public void onMenuOptionClicked(MenuOptionClicked event)
 			lootRecordType = LootRecordType.EVENT;
 			takeInventorySnapshot();
 		}
+
+		if (event.getMenuOption().equals("Open") && SHADE_CHEST_OBJECTS.containsKey(event.getId()))
+		{
+			eventType = SHADE_CHEST_OBJECTS.get(event.getId());
```

Comment:
As written, this would make every chest of different key color recorded in a different loot group, which is kind of weird. I'd expect users to care only about the chest color, not about the key color.